security/aws/aws_config_enabled/aws_config_enabled.pt

```diff
@@ -16,6 +16,8 @@ info(
   benchmark_control: "3.5",
   benchmark_version: "1.4.0",
   cis_controls: "[\"1.1v8\", \"1.4v7\", \"11.2v7\", \"16.1v7\"]",
+  hide_skip_approvals: "true",
+  
```

Comment:
Extraneous empty line should be removed

Reply:
applied at https://github.com/flexera-public/policy_templates/pull/2830/commits/36db4746fcd3b4771d4e66462013e6892ff82fc5